public/api/conf/1.0/common/overview.md

```diff
@@ -0,0 +1 @@
+The User Information API provides information about the signed-in user, in an OpenID Connect compliant format.
```

Comment:
The directory `/common` should be directly under `/public/api/conf` as opposed to the version-specific directories:
https://confluence.tools.tax.service.gov.uk/display/DTRG/RAML+to+OpenAPI+Specification+%28OAS%29+migration+guide#RAMLtoOpenAPISpecification(OAS)migrationguide-APIOverviewandCommonSections